packages/dialtone-vue2/components/split_button/split_button.vue

```diff
@@ -0,0 +1,282 @@
+      data-qa="dt-split-button-alpha"
+      :size="size"
+      :class="[`d-split-btn__alpha d-split-btn__alpha--${size}`]"
+      :active="alphaActive"
+      :aria-label="alphaAriaLabel"
+      :icon-position="alphaIconPosition"
+      :label-class="alphaLabelClass"
+      :loading="alphaLoading"
+      :assertive-on-focus="assertiveOnFocus"
+      :disabled="disabled"
+      :importance="importance"
+      :kind="kind"
+      @click="(e) => handleClick('alpha', e)"
+    >
+      <template #icon>
+        <!-- @slot Alpha (left) button icon slot -->
+        <slot name="alphaIcon" />
+      </template>
+      <!-- @slot Default content slot -->
+      <slot name="default" />
+    </dt-button>
+    <!-- @slot Omega (right) content slot -->
+    <slot name="omega">
+      <dt-dropdown v-if="!omegaId && $slots.dropdownList">
+        <template #anchor="{ attrs }">
+          <split-button-omega
+            v-bind="{ ...attrs, ...omegaButtonProps }"
+            @click="(e) => $emit('omega-clicked', e)"
+          >
+            <template #icon>
+              <!-- @slot Omega (right) button icon slot -->
+              <slot name="omegaIcon" />
+            </template>
+          </split-button-omega>
+        </template>
+        <template #list>
+          <!-- @slot Built-in dropdown contents -->
+          <slot name="dropdownList" />
+        </template>
+      </dt-dropdown>
+
+      <split-button-omega
+        v-else
+        v-bind="omegaButtonProps"
+        @click="(e) => $emit('omega-clicked', e)"
+      >
+        <template #icon>
+          <!-- @slot Omega (right) button icon slot -->
+          <slot name="omegaIcon" />
+        </template>
+      </split-button-omega>
+    </slot>
+  </span>
+</template>
+
+<script>
+import {
+  DtButton,
+  BUTTON_IMPORTANCE_MODIFIERS,
+  BUTTON_KIND_MODIFIERS,
+  BUTTON_SIZE_MODIFIERS,
+  ICON_POSITION_MODIFIERS,
+} from '@/components/button';
+import SplitButtonOmega from './split_button-omega.vue';
+import { DtDropdown } from '@/components/dropdown';
+
+export default {
+  name: 'DtSplitButton',
+
+  components: {
+    DtButton,
+    SplitButtonOmega,
+    DtDropdown,
+  },
+
+  props: {
+    /**
+     * Determines whether the alpha button should have active styling
+     * default is false.
+     * @values true, false
+     */
+    alphaActive: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Descriptive label for the alpha button
+     */
+    alphaAriaLabel: {
+      type: String,
+      default: null,
+    },
+
+    /**
+     * The position of the icon slot within the alpha button.
+     * @values left, right, top, bottom
+     */
+    alphaIconPosition: {
+      type: String,
+      default: 'left',
+      validator: (position) => Object.keys(ICON_POSITION_MODIFIERS).includes(position),
+    },
+
+    /**
+     * Used to customize the alpha label container
+     */
+    alphaLabelClass: {
+      type: [String, Array, Object],
+      default: '',
+    },
```

Comment:
Wondering if omega should have some of this flexibility

Reply:
Don't think so, omega is actually a slot, so you can pass anything you want there.